### DISCUSSION TITLE
"Frequently Asked Questions" Discussion contains reference to non existing inspection "Reformat"

### DISCUSSION THREAD

**Discussion by Exidex** [JetBrains/Qodana#184] [Ideas]
[FAQ](https://github.com/JetBrains/Qodana/discussions/54)  in section "How to make Qodana check formatting rules for Kotlin?" references "Reformat" inspection which was [renamed](https://youtrack.jetbrains.com/issue/QD-774/Provide-IntelliJ-formatting-check) into "IncorrectFormatting"

**Comment by brichbash** [comment]
Thank you for bringing this up. I've corrected the mistake. Besides, we definitely need to update this section
